integration_tests/network.py

```
@@ -179,7 +182,18 @@ def setup_custom_mantra(
     wait_port=True,
     relayer=cluster.Relayer.HERMES.value,
     genesis=None,
+    chain=None,
 ):
+    assert config.suffix == ".jsonnet"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)